wca-guidelines.md

```diff
@@ -164,7 +164,8 @@ To be more informative, each Guideline is classified using one of the following
 - A1a4+) [REMINDER] If a competitor has accidentally been permitted to exceed the time limit, the time limit must be enforced retroactively, and the judge, competitor, and WCA Delegate should be informed (see [Regulation 1g2](regulations:regulation:1g2)). Judges must always be aware of the time limit for a current attempt (which might depend on previous attempts, in the case of a cumulative time limit).
 - A2c1+) [REMINDER] In the past, score sheets have been used to cover puzzles while leaving some sides exposed. This is no longer permitted.
 - A2d1+) [ADDITION] If a missing scramble signature is discovered before an attempt, the puzzle should immediately be returned to a scrambler for verification and a corresponding signature.
```

Comment:
What @xsrvmy proposed sounds better to me

```suggestion
- A2d1+) [ADDITION] If the scramble signature is found to be missing before an attempt the puzzle should immediately be returned to a scrambler for verification and a corresponding signature.
```

Reply:
I think it's important to keep the comma, but sounds good!

```suggestion
- A2d1+) [ADDITION] If the scramble signature is found to be missing before an attempt, the puzzle should immediately be returned to a scrambler for verification and a corresponding signature.
```